src/runtimes/docker/utils/dayjs.ts

```diff
@@ -1,54 +1,54 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
Git wants to change the line endings. No idea why, but it's going to keep trying to do it, so may as well just check it in.